create_dict_drug_name_to_url.py

```diff
@@ -0,0 +1,66 @@
+from bs4 import BeautifulSoup
+import urllib
+import json
+
+url = "https://medlineplus.gov/druginfo/drug_Aa.html"
+
+r = urllib.urlopen(url).read()
+soup = BeautifulSoup(r, "html.parser")
+
+def process_string(strng):
+	name = strng[:]
+	# Remove text inside parentheses
```

Comment:
Instead of this loop, check out this thread for an easy way to remove substrings http://stackoverflow.com/questions/8703017/remove-sub-string-by-using-python